transformer/transformer.go

```diff
@@ -370,9 +370,13 @@ func walkForServices(inputPath string, bservices map[string][]plantypes.PlanArti
 			newPlanServices := getPlanArtifactsFromArtifacts(*env.Decode(&newServicesToArtifacts).(*map[string][]transformertypes.Artifact), config)
 			services = plantypes.MergeServices(services, newPlanServices)
 			logrus.Debugf("[%s] Done", config.Name)
-			numfound += len(newServicesToArtifacts)
-			if len(newServicesToArtifacts) > 0 {
-				relpath, _ := filepath.Rel(inputPath, path)
+			numfound += len(newPlanServices)
+			if len(newPlanServices) > 0 {
+				relpath, err := filepath.Rel(inputPath, path)
+				if err != nil {
+					logrus.Errorf("failed to make the directory %s relative to the input directory %s . Error: %q", path, inputPath, err)
+					continue
```

Comment:
Since it is only for log, we need not continue, but rather print error and proceed to log.

Reply:
Even if it is only for log it should not fail.